manifests/pulp/node.pp

```diff
@@ -0,0 +1,105 @@
+# The pulp node configuration
+#
+# In this scenario we assume there is another pulp instance somewhere and this
+# talks to a qpid router somewhere.
```

Comment:
This is technically speaking incorrect. This [connection diagram](https://raw.githubusercontent.com/ehelms/connection_diagram/master/katello.png) may help with wrapping your brain around all the connection points. 

The dispatch router serves as a public interface for services that need to talk to qpid. Generally speaking thats only clients talking to dispatch router which then talks to either Qpid directly (on a server) or talks to another dispatch router (on a smart proxy). Currently Pulp talks directly to qpid over localhost. If we worked on a spread deployment and split out qpid we'd need to consider having Pulp talk to the router but for current setups it does not.